rotkehlchen/chain/ethereum/modules/yearn/vaultsv2.py

```diff
@@ -189,6 +189,8 @@ def get_vaults_history(
                 query_addresses.append(EthAddress(address.lower()))
                 query_checksumed_addresses.append(address)
 
+        # Store events retrieved from graph + database retrieved in a first query
```

Comment:
```suggestion
        # Store events retrieved from graph + database retrieved in a first query
```

nitpick: Comment does not make sense. You use "retrieve" twice and it's confusing. What did you mean to say?

Reply:
I made the improvement suggested and this is not needed anymore. What I meant is that in this dictionary we store the combination of information retrieved from the subgraph and the database